adapters/consumable/consumable.go

```diff
@@ -124,6 +140,29 @@ func (a *ConsumableAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *a
 		body.Url = request.Site.Page     // where the impression will be made
 	}
 
+	gdpr := bidGdpr{}
+
+	if request.Regs != nil && request.Regs.Ext != nil {
+		var regsExt regsExt
+		if err := json.Unmarshal(request.Regs.Ext, &regsExt); err == nil {
+			body.Ccpa = regsExt.UsPrivacy
+
+			if regsExt.Gdpr != nil {
+				applies := *regsExt.Gdpr != 0
+				gdpr.Applies = &applies
+				body.Gdpr = &gdpr
+			}
+		}
+	}
+
+	if request.User != nil && request.User.Ext != nil {
```

Comment:
I'll eventually have a similar `gdpr.ReadPolicy()` at some point, but that's still a few months away. I'll ping you (maybe via a proposed PR?) to streamline this code when that's done. For now, please consider using the shared structs.

Reply:
Will do, please do ping me when `gdpr.ReadPolicy()` is available.

Reply:
Aye.